addons/website/static/src/scss/website.scss

```diff
@@ -395,6 +472,19 @@ font[class*='bg-'] {
         &.btn-xs { @include button-size(.15rem, .45rem, ($font-size-base * .75), 30px); }
     }
 }
+@each $color, $value in ['primary', 'secondary'] {
+    .btn-#{$color} {
+        @if o-website-value('btn-#{$color}-outline') {
+            @extend .btn-outline-#{$color};
+        } @else {
+            @extend .btn-fill-#{$color};
+        }
+        @if o-website-value('btn-#{$color}-flat') {
+            @extend .btn.flat;
+        }
```

Comment:
Is it possible to be outline and flat at the same time?

Reply:
Strangely yes:
- Drop Text snippet
- Use the "/button" command
- Make the new button Primary & Flat
- Go to the themes tab
- Make Primary buttons outline

But still, the primary/secondary can only be one of the three options. I adapted the condition to `else if` the flat option.